ui/lib/core/addon/components/form-field.hbs

```diff
@@ -302,8 +302,9 @@
           autocomplete="off"
           spellcheck="false"
           value={{or (get @model this.valuePath) @attr.options.defaultValue}}
-          onchange={{this.onChangeWithEvent}}
-          onkeyup={{this.handleKeyUp}}
+          {{on "input" this.onChangeWithEvent}}
+          {{on "change" this.onChangeWithEvent}}
```

Comment:
on "change" is redundant here, now that "input" has been added, right?

Reply:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event):
> The change event is fired for [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input), [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select), and [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) elements when the user modifies the element's value. Unlike the [input](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) event, the change event is not necessarily fired for each alteration to an element's value.

It looks like they are *mostly* interchangeable, and we should favor `change` events because historically the [input event isn't always implemented to spec.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event)

Reply:
Yeah that's what I read from the docs, but when it was _just_ on `change` the model wasn't updating properly 🤔